CommonApi/src/main/java/mezz/jei/api/recipe/category/extensions/IGlobalRecipeCategoryExtension.java

```diff
@@ -0,0 +1,67 @@
+package mezz.jei.api.recipe.category.extensions;
+
+import mezz.jei.api.gui.drawable.IDrawable;
+import mezz.jei.api.gui.ingredient.IRecipeSlotsView;
+import mezz.jei.api.helpers.IGuiHelper;
+import mezz.jei.api.recipe.category.IRecipeCategory;
+import net.minecraft.client.gui.GuiGraphics;
+import net.minecraft.network.chat.Component;
+
+import java.util.List;
+
+public interface IGlobalRecipeCategoryExtension<T> {
```

Comment:
This interface should have a doc to explain its intended purpose, and an `@since 15.1.0` so people know what minimum version of JEI to set for their mod.